backend/src/gh.rs

```diff
@@ -250,16 +250,26 @@ impl GitHubClient {
         base_branch: &str,
         pr_title: &str,
         pr_description: &str,
+        issue_numbers: Option<Vec<u64>>,
     ) -> Result<String> {
         // Parse the repository name from self.repo_url
         let repo_name = self.get_repo_name()?;
 
-        // Prepare the JSON body for the pull request
-        let pr_body = json!({
+        let mut pr_body = pr_description.to_string();
+
+        // If issue numbers are provided, add them to the body
+        if let Some(issues) = issue_numbers {
+            for issue in issues {
+                pr_body.push_str(&format!("\n\nCloses #{}", issue)); // Add "Closes #<issue_number>" for each issue
+            }
+        }
+
+        let pr_body_json = json!({
             "title": pr_title,
             "head": head_branch,
             "base": base_branch,
-            "body": pr_description,
+            "body": pr_body,
+            "labels": ["discord-user:krol430"]
```

Comment:
I'm also a little bit wary of using Github labels to store arbitrary data, simply because then they'll build up over time and I don't want 700+ labels being created that aren't used.

You could probably just include this in the title, or in the body of the PR as a TOML frontmatter header or something, eg:
```
---
key=value
---
We already pull in the TOML crate, so this should be trivial, and hopefully scale well.